src/Php80/Resources/stubs/Stringable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Stringable
+{
+    /**
+     * @return string
+     */
+    public function __toString();
```

Comment:
I think this needs a `: string` to be compatible with the RFC right?

Reply:
Nope, it doesn't - and adding it would force a BC break on code that wants to adopt the polyfill. See the RFC, this is discussed.

Reply:
This one runs on php7, the stub on php8, where the BC layer is in place. So yes, you're misunderstanding something :)

Reply:
> This one runs on php7, the stub on php8, where the BC layer is in place. So yes, you're misunderstanding something :)

this ensured by the test `Zend/tests/stringable_automatic_implementation.phpt`. https://github.com/php/php-src/pull/5083/files#diff-f3a64c6266183080afe34570b2b90282

@nicolas-grekas does `symfony/polyfill` already test compliance with the `php/php-src` Test Suite? It should be to integrate, partial usage of their test suite to cover `symfony/polyfill` easy to run those tests over the ployfill code. Let me know if this is worth a shot and i could help.

Reply:
We don't do it systematically. In this specific case, the behavior on php8 cannot be replicated on php7. But that does invalidate the polyfill, which still works, in an opt in way.